### PR TITLE
Python3 compatibility fix

### PR DESCRIPTION
**Pull request by LaurieCheers-unity**

### CODE REVIEW

**Comment by LaurieCheers-unity** [comment]
Turns out the file doesn't even use StringIO so I just removed the import

File: tcp_endpoint/src/tcp_endpoint/RosSubscriber.py [hunk 1]
```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env python
-import StringIO
 
+try:
+    from StringIO import StringIO ## for Python 2
+except ImportError:
+    from io import StringIO ## for Python 3
+
```
Comment (hyounesy) [inline tcp_endpoint/src/tcp_endpoint/RosSubscriber.py]: I am an expert in Python 2-3 compatibility, but [based on this](http://python-future.org/compatible_idioms.html?highlight=stringio#stringio), this line should work for both Python 2 and 3 and you probably won't need the one above.
```python
# Python 2 and 3:
from io import BytesIO     # for handling byte strings
from io import StringIO    # for handling unicode strings
```

Comment (LaurieCheers-unity) [reply tcp_endpoint/src/tcp_endpoint/RosSubscriber.py]: Maybe we can use that in future, when there's a file that actually needs to import StringIO... Just removing the import line seems much easier. :-)

**Review by mpinol — APPROVED** [review]

File: tcp_endpoint/src/tcp_endpoint/RosSubscriber.py (hunk 1 above)
Comment (mpinol) [reply tcp_endpoint/src/tcp_endpoint/RosSubscriber.py]: Just to round this conversation out. https://github.com/Unity-Technologies/ROS-TCP-Endpoint/blob/main/tcp_endpoint/src/tcp_endpoint/RosTCPClientThread.py#L4 Is the only file that is using BytesIO

Comment (hyounesy) [reply tcp_endpoint/src/tcp_endpoint/RosSubscriber.py]: agree. I only noticed that you removed it in your next commit after making the comment 😄

**Review by hyounesy — APPROVED** [review]

**Pull request merged**